homeassistant/strings.json

```diff
@@ -56,8 +56,9 @@
       "abort": {
         "single_instance_allowed": "Already configured. Only a single configuration possible.",
         "already_configured_account": "Account is already configured",
-        "already_configured_service": "Service is already configured",
         "already_configured_device": "Device is already configured",
+        "already_configured_location": "Location is already configured",
```

Comment:
I'm not sure if it is better to use `Location is already configured` or `Service for this location is already configured`.

Reply:
I think it's ok like this.